api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -76,6 +76,17 @@
           "https://www.googleapis.com/auth/userinfo.email",
           "https://www.googleapis.com/auth/cloud-billing");
 
+  // All options are defined in this document:
+  // https://docs.google.com/document/d/1YS95Q7ViRztaCSfPK-NS6tzFPrVpp5KUo0FaWGx7VHw/edit#
+  public static final List<String> FIRECLOUD_GET_WORKSPACE_REQUIRED_FIELDS =
+      Arrays.asList(
```

Comment:
nit: We can use an immutable class like `ImmutableList` from Guava or `List.of()` from Java 9. Not sure what Java version we're using so may have to go with the former.